ReactWindows/ReactNative/UIManager/BaseViewManager.cs

```diff
@@ -415,6 +518,58 @@ private static void SetOverflowVisible(TFrameworkElement element)
             element.Clip = null;
         }
 
+        private static async Task<JObject> GetDataTransferInfo(DataPackageView data, bool drop = false)
+        {
+            var files = new JArray();
+            var items = new JArray();
+            var types = new JArray();
+
+            if (data.Contains(StandardDataFormats.StorageItems))
+            {
+                foreach (var item in await data.GetStorageItemsAsync())
+                {
+                    try
+                    {
+                        var file = item as StorageFile;
+
+                        if (file != null)
+                        {
+                            var props = await file.GetBasicPropertiesAsync();
+                            var type = file.ContentType;
+                            var path = drop ? FutureAccessList.Add(file).ToString() : "";
+
+                            files.Add(new JObject
+                            {
+                                { "name", file.Name },
+                                { "size", props.Size },
+                                { "type", type },
+                                { "uri", path }
+                            });
+
+                            items.Add(new JObject
+                            {
+                                { "kind", "file" },
+                                { "type", type }
+                            });
+
+                            types.Add(type);
+                        }
+                    }
+                    catch (Exception ex)
+                    {
+                        System.Diagnostics.Debug.WriteLine("GetDataTransferInfo: " + ex);
```

Comment:
>System.Diagnostics.Debug.WriteLine("GetDataTransferInfo: " + ex); [](start = 24, length = 65)

When might this happen?

Reply:
Indeed, and should we put some more general "soft exception" handling mechanism in so there's only one place to decide `Debug.WriteLine` vs. something like a JS module that displays a yellow box, for example.

---
In reply to: [181926381](https://github.com/Microsoft/react-native-windows/pull/1549#discussion_r181926381) [](ancestors = 181926381)

Reply:
I don't remember the details, but most likely I added this `catch` just in case (I thought it's not reasonable to crash the app only because of a failure to process dropped items). Removing the `catch` for now.